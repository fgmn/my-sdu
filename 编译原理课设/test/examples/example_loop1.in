procedure f1;
	procedure f2;
		procedure f3;
		begin
		end;
	begin
	end;
begin
end;
begin
end.