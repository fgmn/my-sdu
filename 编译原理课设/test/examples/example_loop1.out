PROGRAM(SUBPROG(PROCEDUREDECLARE(PROCEDUREHEAD(PROCEDURE,F1,;),SUBPROG(PROCEDUREDECLARE(PROCEDUREHEAD(PROCEDURE,F2,;),SUBPROG(PROCEDUREDECLARE(PROCEDUREHEAD(PROCEDURE,F3,;),SUBPROG(SENTENCE(COMBINED(BEGIN,SENTENCE(EMPTY),END))),;),SENTENCE(COMBINED(BEGIN,SENTENCE(EMPTY),END))),;),SENTENCE(COMBINED(BEGIN,SENTENCE(EMPTY),END))),;),SENTENCE(COMBINED(BEGIN,SENTENCE(EMPTY),END))),.)
